### DISCUSSION TITLE
App stuck “offline”

### DISCUSSION THREAD

**Discussion by 0000011110110111** [mastodon/mastodon#26067] [General]
The app is stuck “offline” so no conversion threads are loading. Every other app on my phone is working fine. I’ve restarted both the app & my phone, all to no avail. Any ideas? Thx.
![image](https://github.com/mastodon/mastodon/assets/43657564/b2e2007b-f019-4870-a431-821d7576df51)

**Comment by renchap** [comment]
Hi,

There is an open issue in the iOS app repository about this, please check there for further progress on the issue: https://github.com/mastodon/mastodon-ios/issues/898
